docs/getting_started.md

```diff
@@ -39,25 +36,58 @@ traces.
 -   Use open source libraries with OpenTracing integration to get
     vendor-neutral instrumentation for free
 
-### Prerequisites
+###Prerequisites
 
 -   You need Go 1.7 or higher installed on your machine.
 -   Requires a running Jaeger backend to view the traces.
-    -   See [Running Jaeger backend in docker](#running-jaeger-in-docker-container)
+    -   See [Running Jaeger backend in docker](#docker)
 
-### Installation
+###Running 
 
-```shell 
+```bash
 go get github.com/uber/jaeger
+cd $GOPATH/src/github.com/uber/jaeger
 make install_examples
+cd examples/hotrod
+go run ./main.go all
 ```
 
-### Running
+Then navigate to `http://localhost:8080`
 
-```shell
-cd examples/hotrod
-go run ./main.go all
+##Client Libraries
+
+| Language | Library                                                      | 
+| ---------|--------------------------------------------------------------|
+| go       | [jaeger-go](https://github.com/uber/jaeger-client-go)        |
+| java     | [jaeger-java](https://github.com/uber/jaeger-client-java)    |
+| node     | [jaeger-node](https://github.com/uber/jaeger-client-node)    |
+| python   | [jaeger-python](https://github.com/uber/jaeger-client-python)|
+
+
+For a deeper dive into how to instrument a Go service, look at [Tracing HTTP request latency](https://medium.com/@YuriShkuro/tracing-http-request-latency-in-go-with-opentracing-7cc1282a100a).
+
+##Running individual components
+Individual components can be run from source. They have their `main.go` in the `cmd` folder. 
+
+For e.g., to run the agent, you'll have to do the following
+
+```bash
+go get github.com/uber/jasger
+cd $GOPATH/src/github.com/uber/jaeger
+make install
+cd agent
+go run ./main.go 
 ```
 
-Then open <http://127.0.0.1:8080>
+##Migrating from Zipkin
+Jaeger's agent and collector can also accept [Zipkin](http://zipkin.io/) [Spans](https://github.com/openzipkin/zipkin-api/blob/master/thrift/zipkinCore.thrift#L381), and transform them to Jaeger's data model before storage. 
+
+Both of them support this Zipkin [idl](https://github.com/uber/jaeger-idl/blob/master/thrift/zipkincore.thrift), and expose the following thrift service. 
+```thrift
```

Comment:
Don't think thrift service reference provides much value. We might want to put TODO here